packages/sources/kaiko-test/src/endpoint/trades.ts

```diff
@@ -119,16 +121,30 @@ const httpTransport = new HttpTransport<EndpointTypes>({
     return params.map((param) => {
       const data = res.data.data.filter((x) => x.price !== null)
       if (data.length === 0) {
+        const errorMessage = `Kaiko is not returning any price data for ${param.base}/${param.quote}, likely due to too low trading volume for the requested interval. This is not an issue with the external adapter.`
```

Comment:
Let's clarify what the requested interval is here, make it really explicit that we're adding this parameter. It's optional in Kaiko's API, and it always comes up because when you test requesting the source directly if you omit the timeframe it'll pull very old trades

Reply:
Good call! Added in the latest commit